test/extensions/common/wasm/wasm_vm_test.cc

```diff
@@ -49,7 +49,16 @@ class BaseVmTest : public testing::Test {
   Stats::ScopeSharedPtr scope_;
 };
 
-TEST_F(BaseVmTest, NoRuntime) { EXPECT_EQ(createWasmVm(""), nullptr); }
+TEST_F(BaseVmTest, UnspecifiedRuntime) {
+  auto wasm_vm = createWasmVm("");
+  // Envoy is built with "--define wasm=disabled", so no Wasm engine is available
+  if (getFirstAvailableWasmEngineName().empty()) {
+    EXPECT_TRUE(wasm_vm.get() == nullptr);
+  } else {
+    EXPECT_TRUE(wasm_vm.get() != nullptr);
```

Comment:
ASSERT_TRUE -- that way if this line fails the test fails, but doesn't crash on the next line.